server/app/lib/services/elasticsearch/index.js

```diff
@@ -0,0 +1,88 @@
+import elasticsearch from "elasticsearch";
+import tabify from "./tabify";
+import { getUsedParameters } from "../../utils/configurations";
+
+var client = null;
+let config = function () {
+    return {
+        host: process.env.APP_ELASTICSEARCH_HOST ? process.env.APP_ELASTICSEARCH_HOST : null,
```

Comment:
currentConfig.host = state.ES.get(ActionKeyStore.ES_HOST) || process.env.REACT_APP_ELASTICSEARCH_HOST;

We are missing to parse the first param, arnt we?